packages/actor-query-operation-path-zero-or-more/lib/ActorQueryOperationPathZeroOrMore.ts

```diff
@@ -34,36 +34,64 @@ export class ActorQueryOperationPathZeroOrMore extends ActorAbstractPath {
     if (sVar && oVar) {
       // Get all the results of subjects, then fill in first variable for those
       const predVar = this.generateVariable(path);
-      const single = ActorAbstractPath.FACTORY.createUnion(
-        ActorAbstractPath.FACTORY.createPattern(path.subject, predVar, path.object, path.graph),
-        ActorAbstractPath.FACTORY.createPattern(path.object, predVar, path.subject, path.graph),
-      );
+      const single = ActorAbstractPath.FACTORY.createPattern(path.subject, predVar, path.object, path.graph);
       const results = ActorQueryOperation.getSafeBindings(
         await this.mediatorQueryOperation.mediate({ context, operation: single }),
       );
       const subjectString = termToString(path.subject);
       const objectString = termToString(path.object);
 
-      const subjects: Set<string> = new Set();
+      const entities: Set<string|undefined> = new Set();
```

Comment:
Why do we want undefined in our set?

Reply:
It's because that's is the returntype of termToString. I can cast it before I put it in the set, if that's better?

Reply:
The return type should always be `string` for any given defined term: https://github.com/rubensworks/rdf-string.js/blob/master/lib/TermUtil.ts#L22